pkg/controller/stacks/install/stackinstall.go

```diff
@@ -237,6 +250,31 @@ func (h *stackInstallHandler) update(ctx context.Context) (reconcile.Result, err
 	return reconcile.Result{}, nil
 }
 
+func (h *stackInstallHandler) delete(ctx context.Context) (reconcile.Result, error) {
+	labels := map[string]string{
+		labelInstallNamespace: h.ext.GetNamespace(),
+		labelInstallName:      h.ext.GetName(),
+		labelInstallUID:       string(h.ext.GetUID()),
+	}
+	list := &apiextensionsv1beta1.CustomResourceDefinitionList{}
+	if err := h.kube.List(ctx, list, client.MatchingLabels(labels)); err != nil {
+		return fail(ctx, h.kube, h.ext, err)
+	}
+
+	for i := range list.Items {
+		if err := h.kube.Delete(ctx, &list.Items[i]); err != nil {
```

Comment:
we should ignore "err not found" in case it was already deleted.

Reply:
Added in 66f7584a